docs/demos/first_steps.md

```diff
@@ -33,21 +33,21 @@ The `osm` command-line tool contains everything needed to install and configure
 The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
 
 ### For GNU/Linux or macOS
-Download and unzip the 64-bit [GNU/Linux](https://github.com/openservicemesh/osm/releases/download/v0.8.0/osm-v0.8.0-linux-amd64.tar.gz)
+Download and unzip the 64-bit [GNU/Linux](https://github.com/openservicemesh/osm/releases/download/v0.9.0/osm-v0.9.0-linux-amd64.tar.gz)
```

Comment:
We should either be referring to v0.9.0-rc.1 throughout or waiting to make these changes until v0.9.0.

Reply:
Prefer updating it to v0.9.0-rc.1